spam-n-eggs/laravel-mysqlite
# Trying to fix packagist problems

# Details

Trying to fix the packagist problems with our json files.

# Steps to test changes

1. Run "composer validate" to make sure the files are valid.
1. Run "vendor/bin/phpunit" to run all the phpunit tests.
1. Make sure all the tests pass and everything is working fine.

# Issues Addressed

[CU-mryuk7](https://app.clickup.com/t/mryuk7) - [Laravel-MySQLite] Packagist.org is complaining that the composer.json is corrupt
